.releaseindex.json

```diff
@@ -1,6 +1,6 @@
 {
   "WindowsClassic":  "Libation\\.\\d+\\.\\d+\\.\\d+-win(dows)?-classic\\.zip",
   "WindowsAvalonia": "Libation\\.\\d+\\.\\d+\\.\\d+-win(dows)?-chardonnay\\.zip",
-  "LinuxAvalonia":   "Libation\\.\\d+\\.\\d+\\.\\d+-linux-chardonnay",
-  "MacOSAvalonia":   "Libation\\.\\d+\\.\\d+\\.\\d+-macos-chardonnay"
+  "LinuxAvalonia":   "Libation\\.\\d+\\.\\d+\\.\\d+-linux-chardonnay\\.deb",
+  "MacOSAvalonia":   "Libation\\.app-x64-\\d+\\.\\d+\\.\\d+\\.tgz"

```

Comment:
Can we still have 'macos' in here somewhere? I assume this is the package name that will be seen by downloaders and the app and tgz don't make it immediately obvious that this is for mac.

Reply:
Done!